packages/components-react/src/heading/Heading.tsx

```diff
@@ -0,0 +1,47 @@
+import { Heading as UtrechtHeading } from '@utrecht/component-library-react/dist/css-module';
+import { HTMLAttributes, PropsWithChildren, ReactNode } from 'react';
+
+type HeadingLevels = 1 | 2 | 3 | 4 | 5 | 6;
+
+export interface LuxHeadingProps extends HTMLAttributes<HTMLHeadingElement> {
+  appearance?: HeadingLevels;
+  level: HeadingLevels;
+}
+
+export const LuxHeading = ({
+  children,
+  level = 6,
```

Comment:
I don't think we should set a default level. The `level` here is too important for the browser/end user so we need to force the implementing developer to make a conscious choice